lib/rules/lint-quotes.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const Rule = require('./base');
+const createErrorMessage = require('../helpers/create-error-message');
+
+module.exports = class Quotes extends Rule {
+  parseConfig(config) {
+    let configType = typeof config;
+
+    switch (configType) {
+    case 'boolean':
+      if (!config) {
+        return false;
+      }
+      break;
+    case 'string':
+      if (['double', 'single'].indexOf(config) > -1) {
+        return config;
+      }
+      break;
+    case 'undefined':
+      return false;
+    }
+
+    let errorMessage = createErrorMessage(this.ruleName, [
+      '  * "double" - requires the use of double quotes wherever possible',
+      '  * "single" - requires the use of single quotes wherever possible'
+    ], config);
+
+    throw new Error(errorMessage);
+  }
+
+  visitor() {
+    let badChar;
+    let message;
+    switch (this.config) {
+    case 'double':
+      badChar = '\'';
```

Comment:
Thoughts on using a template string here instead to avoid the escaping?

Reply:
I think I got a linting error when I tried that.

Reply:
yep "Strings must use singlequote."